web/packages/teleport/src/HeadlessSSO/HeadlessSSO.tsx

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright 2023 Gravitational, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import React, {useEffect, useState} from 'react';
+import auth from 'teleport/services/auth';
+import { useParams } from 'teleport/components/Router';
+import HeadlessSsoDialog from 'teleport/components/HeadlessSsoDialog/HeadlessSsoDialog';
+import {CardAccept, CardDenied} from "teleport/HeadlessSSO/Cards";
+
+
+export function HeadlessSSO() {
```

Comment:
In the web code the convention is a bit different than Go.
```suggestion
export function HeadlessSso() {
```

Reply:
I wasn't sure as I used both in different places. I'll fix that 😅